src/Dispatcher.php

```diff
@@ -51,6 +51,9 @@ class Dispatcher
     /** @var Receipt Last Receipt */
     protected $lastReceipt;
 
+    /** @var array of last warnings */
+    protected $lastWarnings;
```

Comment:
Pokud je propertie array, nemělo by to být spíše `protected $lastWarnings = array();` ?